x/crosschain/keeper/keeper.go

```diff
@@ -53,7 +53,7 @@ func NewKeeper(
 		paramstore:         paramstore,
 		authKeeper:         authKeeper,
 		bankKeeper:         bankKeeper,
-		zetaObserverKeeper: zetaObserverKeeper,
+		ZetaObserverKeeper: zetaObserverKeeper,
```

Comment:
why not add a GetZetaObserverKeeper() function  ?

Reply:
reversed the renaming. 
now changed from reduced from 45->34.